platforms/unix/plugins/SecurityPlugin/sqUnixSecurity.c

```diff
@@ -213,7 +213,8 @@ ioInitSecurity(void)
   if (imagePathLen)
     strncpy(secureUserDirectory, imageName, imagePathLen);
   else {
-    getwd(secureUserDirectory);
+/*@@MUSL@@*/
+    getcwd(secureUserDirectory, imagePathLen);
```

Comment:
Does it work? We're passing `imagePathLen==0` here.
Shouldn't it be the buffer size, `MAXPATHLEN`?